timetracker/account/views.py

```diff
@@ -1,10 +1,11 @@
 import logging
-
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.contrib.auth.views import LoginView as BaseLoginView
 from django.views.generic import TemplateView
-
-
+from django.contrib.auth import login, authenticate
+from django.shortcuts import redirect
+from django.views import generic
+from account import forms
```

Comment:
Typically, imports are ordered alphabetically with line breaks separating the standard library from third party libraries from custom modules. So the above would look like:

```python
import logging

from django.contrib.auth import authenticate, login
from django.contrib.auth.mixins import LoginRequiredMixin
from django.contrib.auth.views import LoginView as BaseLoginView
from django.shortcuts import redirect
from django.views.generic import TemplateView

from account import forms
```